subiquity/server/apt.py

```diff
@@ -73,7 +73,7 @@ def apt_sourceparts_files(mp: Mountpoint) -> List[str]:
     root = mp.pp()
     sources_list_d = root / "etc/apt/sources.list.d"
     paths = list(sources_list_d.glob("*.list")) + list(sources_list_d.glob("*.sources"))
-    return [str(p.relative_to(root)) for p in paths]
+    return [str(p.relative_to(root)) for p in paths if p.name != "cdrom.sources"]
```

Comment:
we can choose any name we want right?  I think I want to choose a more specific name here than "cdrom.sources", which should also help the potential conflicts case.

Also why is the file currently called "cdrom.sources" handled special here?  If we do need that would you add a note about it in the comment?

Reply:
I guess it doesn't strictly have to be coordinated. We could identify the sources file by running `apt-get indextargets --format '$(SOURCESENTRY)' "Base-URI: file:/cdrom/dists/$SERIES"` or something like that.